lib/editor/components/ScheduleExceptionForm.js

```diff
@@ -212,14 +226,26 @@ export default class ScheduleExceptionForm extends Component<Props> {
             )
             : null
           }
+          {exemplar === EXCEPTION_EXEMPLARS.EXCEPTION_SERVICE &&
```

Comment:
This feels brittle but I'm realizing this is just how datatools does things... please ignore previous comment

Reply:
Brittle bc the constant could change and we would lose the component?